src/components/v5/common/CompletedAction/partials/MeatballMenu/hooks.tsx

```diff
@@ -0,0 +1,63 @@
+import { ArrowSquareOut, Repeat } from '@phosphor-icons/react';
+import React, { useMemo } from 'react';
+
+import { DEFAULT_NETWORK_INFO } from '~constants';
+import { useActionSidebarContext } from '~context/ActionSidebarContext/ActionSidebarContext.ts';
+import TransactionLink from '~shared/TransactionLink/TransactionLink.tsx';
+import { formatText } from '~utils/intl.ts';
+import { type MeatBallMenuItem } from '~v5/shared/MeatBallMenu/types.ts';
+
+export const useGetItems = ({
```

Comment:
as mentioned above, I think we don't need this hook, we wouldn't even be thinking of it if the `MeatballMenu` component accepted children instead of arrays with JSX